### ISSUE TITLE
Save/cancel button when editing API title embeded in Syndesis

### ISSUE THREAD

**Issue by zregvart** [Apicurio/apicurio-studio#477]
When the Apicurio editor is embedded in Syndesis there are two Save/Cancel button pairs present. Perhaps Apicurio one's can be replaced with the checkmark/xmark to avoid any confusion.

This is how it looks when embedded:
![screenshot_2018-10-18 syndesis](https://user-images.githubusercontent.com/1306050/47154224-fe41cd80-d2e1-11e8-8712-b59495646ef4.png)

This is how save/cancel controls look on other inputs:
![screenshot_2018-10-18 syndesis 1](https://user-images.githubusercontent.com/1306050/47154328-55e03900-d2e2-11e8-8f34-40e1fedb6b11.png)

**Comment by EricWittmann** [comment]
I'd like @Mgranfie to respond to this.  :)  Monica - any thoughts?  Zoran - can you provide a larger screenshot for more context?

Note: this only happens when the user chooses to edit the **title** of the API.  The Save/Cancel buttons are specifically for saving changes made to the title.  This is not an argument against changing them - just providing background and clarification of current state.

**Comment by Mgranfie** [comment]
@zregvart @EricWittmann good catch! It would be helpful to see a full screenshot of the use case here just to confirm direction. If this is just to save the edits on the API title, I am thinking that the checkmark and x will work fine in this case, but would like to see the full screen use case before confirming..

**Comment by zregvart** [comment]
There are two places where we embed Apicurio. When user edits OpenAPI specification that the user has uploaded/pointed via URL to create an API client:

![screenshot_2018-10-18 syndesis](https://user-images.githubusercontent.com/1306050/47156551-a8702400-d2e7-11e8-9d70-7e885c169994.png)

And when the user edits/creates from scratch an OpenAPI specification that is to be implemented in Syndesis:
![screenshot_2018-10-18 syndesis 1](https://user-images.githubusercontent.com/1306050/47156581-bcb42100-d2e7-11e8-81df-e87bc4ebf628.png)

There are some subtle differences, like the label next to the back link, but those two use cases are almost identical.

**Comment by EricWittmann** [comment]
Just to be clear - you ONLY see the Apicurio Save/Cancel buttons when you click on the API title to edit it, right?  Those buttons shouldn't be there when you first open the API for editing.

**Comment by Mgranfie** [comment]
Yes, I would agree that the check and x are the best solution in this use case. If I am remembering correctly Dongni and I landed on this as a solution a while back? It also looks like the case of save/cancel modality while the api title is being edited has been addressed with button states, which is great. That said, I am OK with moving forward with replacing the save/cancel buttons with the  check and X. @EricWittmann I like this solution overall, even in the stand alone case, but can't remember exactly why we came to the decision to use Save/Cancel...? :)

**Comment by EricWittmann** [comment]
Not sure, but I'm happy to change it.

**Comment by zregvart** [comment]
> Just to be clear - you ONLY see the Apicurio Save/Cancel buttons when you click on the API title to edit it, right? Those buttons shouldn't be there when you first open the API for editing.

That is correct.

**Comment by EricWittmann** [comment]
I've updated the UI in Apicurio.  For reference see:  https://github.com/Apicurio/apicurio-studio/issues/479

I'll close **this** issue once a new release has been done and an update PR has been sent to syndesis.
